bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -58,25 +58,24 @@
 				<description>JSON config that allows to modify the media state detection strategy for each app. Refer to the binding
 					documentation.</description>
 			</parameter>
-			<parameter name="volumeSettingKey" type="text" required="true">
+			<parameter name="volumeSettingKey" type="text">
 				<label>Volume Setting Key</label>
 				<description>Settings key for android versions where the volume level is gathered using the 'settings' cli
 					(>=android 11).</description>
 				<default>volume_music_hdmi</default>
 				<options>
-					<option value="volume_bluetooth_sco">volume_bluetooth_sco</option>
-					<option value="volume_music">volume_music</option>
-					<option value="volume_music_bt_a2dp">volume_music_bt_a2dp</option>
-					<option value="volume_music_hdmi">volume_music_hdmi</option>
-					<option value="volume_music_headphone">volume_music_headphone</option>
-					<option value="volume_music_headset">volume_music_headset</option>
-					<option value="volume_music_headset">volume_music_headset</option>
-					<option value="volume_music_usb_headset">volume_music_usb_headset</option>
-					<option value="volume_system">volume_system</option>
+					<option value="volume_bluetooth_sco">volume bluetooth sco</option>
+					<option value="volume_music">volume music</option>
+					<option value="volume_music_bt_a2dp">volume music bt a2dp</option>
+					<option value="volume_music_hdmi">volume music hdmi</option>
+					<option value="volume_music_headphone">volume music headphone</option>
+					<option value="volume_music_headset">volume music headset</option>
+					<option value="volume_music_usb_headset">volume music usb headset</option>
+					<option value="volume_system">volume system</option>
```

Comment:
You have also to adjust again the i18n entries.

Reply:
ty!